test/jest/unit/lib/commands/fix/fix.spec.ts

```diff
@@ -110,6 +114,36 @@ describe('snyk fix (functional tests)', () => {
         dryRun: true,
         quiet: true,
       });
+      expect(addAnalyticsSpy).toHaveBeenCalledTimes(8);
+      expect(addAnalyticsSpy.mock.calls[0]).toEqual([
+        'snykFixFailedProjects',
+        0,
+      ]);
```

Comment:
nice work! this works, however, the more robust way to write a test if the order of execution doesn't matter might be
```suggestion
      expect(addAnalyticsSpy).toHaveBeenCalledWith('snykFixFailedProjects', 0);
```

the benefit here is that if the order of execution changes in the implementation, this test will still pass which I think is desired. This will prevent future test failures for wrong reason (implementation works it's just in different order of statements that's changed)

what do you think?

Reply:
i like it! forgot you can call it like that. Easier to read too. Will push an update